src/status_im/ui/screens/chat/views.cljs

```diff
@@ -299,6 +344,8 @@
                     :on-layout (fn [e]
                                  (re-frame/dispatch [:set :layout-height (-> e .-nativeEvent .-layout .-height)]))}
         [chat-toolbar current-chat public? modal?]
+        [react/text {:style {:height 14}}
```

Comment:
leftover?

Reply:
in current build when you open chat screen first time `limit enabled: true` is shown which means it is rendered with new limits, the next time `limit enabled: true` is shown -  that's an old version. Just for testing

Reply:
do you plan to remove this after testing?

Reply:
Sure